### PR TITLE
Support for Kia Niro PHEV 2020

### PR DESCRIPTION
**Pull request by Maxwell-Rosen**
***** Car port *****
I included the appropriate fingerprints for the Kia Niro PHEV 2020. I installed it on my comma 3 and the software works great. It has the issue of no steering control below 32 mph, but that is the only thing. I'm not too sure how to update the docs and such. The checklist seems beyond my capabilities. The changes I made are included in the values.py which I modified. I only added a few lines to that file.

**Checklist**
- [ yes, no] added entry to CarInfo in selfdrive/car/*/values.py and ran `selfdrive/car/docs.py` to generate new docs
- [ ?] test route added to [routes.py](https://github.com/commaai/openpilot/blob/master/selfdrive/car/tests/routes.py)
- [ ?] route with openpilot:
- [ ?] route with stock system:

### CODE REVIEW

**Comment by sshane** [comment]
Can you provide a dongle id?

**Comment by mrquell** [comment]
I belive my dongle ID is ddae15faeff8ee16

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -1202,22 +1202,32 @@ class Buttons:
...
     ],
     (Ecu.transmission, 0x7e1, None): [
       b"\xf1\x816U3J2051\x00\x00\xf1\x006U3H0_C2\x00\x006U3J2051\x00\x00PDE0G16NS2\xf4'\\\x91",
       b'\xf1\x816U3J2051\x00\x00\xf1\x006U3H0_C2\x00\x006U3J2051\x00\x00PDE0G16NS2\x00\x00\x00\x00',
       b'\xf1\x816U3H3051\x00\x00\xf1\x006U3H0_C2\x00\x006U3H3051\x00\x00PDE0G16NS1\x00\x00\x00\x00',
       b'\xf1\x816U3H3051\x00\x00\xf1\x006U3H0_C2\x00\x006U3H3051\x00\x00PDE0G16NS1\x13\xcd\x88\x92',
+      b'\xf1\x006U3H1_C2\x00\x006U3J9051\x00\x00PDE0G16NL2&[\xc3\x01',
+      b'\xf1\x816U3J9051\x00\x00\xf1\x006U3H1_C2\x00\x006U3J9051\x00\x00PDE0G16NL2&[\xc3\x01',
+      b'PDE0G16NL2',
+      b'\xf1\x896U3J9051\x00\x00\xf1\x82PDE0G16NL2',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:1217]:
```suggestion
```

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -1202,22 +1202,32 @@ class Buttons:
     (Ecu.engine, 0x7e0, None): [
       b'\xf1\x816H6F4051\x00\x00\x00\x00\x00\x00\x00\x00',
       b'\xf1\x816H6D1051\x00\x00\x00\x00\x00\x00\x00\x00',
+      b'\xf1\x816H6F6051\x00\x00\x00\x00\x00\x00\x00\x00',
+      b'DEDAS6PB0SF0\x00\x00\x00\x00',
+      b'\xf1\x82DEDAS6PB0SF0\x00\x00\x00\x00',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:1207]:
```suggestion
```

**Comment by sshane** [comment]
Can you confirm which harness you used for your vehicle, as well as what speeds the car enables, and steers at?

**Comment by mrquell** [comment]
It is a Hyundai D harness and there is no lateral control below 32 mph, but there is speed control at all speeds

**Comment by sshane** [comment]
So you can engage at any speed, right? Currently, openpilot limits your steering speed to 32 mph, but it may be lower. Can you try to remove these two lines on your device or a fork, and let me know if it can steer to a stop?

https://github.com/commaai/openpilot/blob/3d98cb72c08ec5d9b9ac5645ea9effecadd6bc38/selfdrive/car/hyundai/interface.py#L137-L138

**Pull request merged**